### ISSUE TITLE
sound for entering the lobby

### ISSUE THREAD

**Issue by andy5995** [ZetaGlest/zetaglest-data#44]
Need a new sound when entering the lobby.

I was thinking along the lines of a two tone, fade-in,fade-out sound, that lasts for about 2 to 3 seconds.

**Comment by adam-kuhn** [comment]
i'll give this a whirl tomorrow

**Comment by andy5995** [comment]
Thanks @adam-kuhn .

I must correct myself a little bit. There is no sound for entering the lobby! xD

The sound happens when a new server becomes available. When a user enters the lobby, the program basically sees new servers that are available. Unless there are no servers available then no sound is played.

So.. a sound effect for a server becoming available? I have nothing in mind yet about how that should sound. But a short, simple sound; a sound that can get the attention of someone who may not be next to the computer.

**Comment by adam-kuhn** [comment]
Ok so you'll need me to create the file for this? Maybe I'll tackle the Battering ram sound first, to git familiar with your base

**Comment by andy5995** [comment]
Yes. And please do work on whatever you're most comfortable with. Also, feel free to drop by our Discord server if you have any questions.
